### DISCUSSION TITLE
希望有一份运筹优化的实战项目

### DISCUSSION THREAD

**Discussion by learning-ontheway** [datawhalechina/DOPMC#202] [问题求助]
看见datawhale几年前开始过一个类似的项目但没有继续做下去，觉得挺可惜的。
https://datawhalechina.github.io/undingable-optimization/#/README
另外对运筹优化的实战更感兴趣，如果可以在理论的基础上加入更多实践内容就更好啦。

**Comment by liu-yang-maker** [comment]
Hello, Ariaayoo, I'm honored for you notice the [undeniable optimization repo](https://datawhalechina.github.io/undingable-optimization/#/README) and I'm the originator for the topic. If you are interested in learning more about combinatorial optimization and discussing, you can consider discuss more content with DOPMC and give me some suggestions.

**Reply by learning-ontheway** [reply]
I think your structure of this project is quite good. When I was in the undergraduate study I used this textbook ‘Operation Research’ (https://book.douban.com/subject/19896239/). I think this project can also refer to its structure. But this book only introduce the theory and thoughts of this area and don't have any codes and practical examples (For example where can these theory and thoughts used in other fields?  It would be more attractive if we can also conclude a more understandable learning document of Gurobi Module combining with small and big examples (just like joyful pandas and wonderful sql). 
This book can also give you some enlightenments: 运筹优化常用模型、算法及案例实战——Python+Java实现 (https://book.douban.com/subject/36278471/)
